roles/ipabackup/README.md

```diff
@@ -46,6 +46,7 @@ Requirements
 **Node**
 * Supported FreeIPA version (see above)
 * Supported distribution (needed for package installation only, see above)
+* Python interpreter accessible at `/usr/bin/python` or ansible special variable `ansible_python_interpreter` set for the host you wish to backup
```

Comment:
I'm not sure this note should be added to the README.

Reply:
What is the rationale against it ?
A maybe more satisfying, though convoluted, alternative would be to add a stat task before the collection of the backup dir that verify the existence of /usr/bin/python or ansible_python_interpreter when it's set ? This way the role would throw an error that could be a little more understandable.

Reply:
The rationale is that `ansible_python_interpreter` is (or should be) always set, either as injected fact or in ansible_fact dict.
By changing ansible_python_interpreter you don't only change the interpreter for this task, but for the whole role execution, as it is set for the host.

Reply:
Oh of course, forgot python was required to execute modules on host. I'm removing it from the readme.